src/app/components/ATIAnalytics/canonical/index.tsx

```diff
@@ -58,8 +68,9 @@ const CanonicalATIAnalytics = ({ pageviewParams }: ATIAnalyticsProps) => {
 
   return (
     <>
-      {addOperaMiniExtremeScript(atiPageViewUrlString)}
-      {renderNoScriptTrackingPixel(atiPageViewUrl, isLite)}
+      {isLite && addLiteScript(atiPageViewUrlString)}
+      {!isLite && addOperaMiniExtremeScript(atiPageViewUrlString)}
```

Comment:
Just thinking out loud here... what happens if the user views the lite site on opera mini? 😰 

I think we're OK, but just wanted to check.

Reply:
It should be ok. The Lite script is basically identical to the Opera Mini script, so they'll get the same behaviour.

Reply:
And there's no danger of analytics being sent twice?

Reply:
This may be something we'll need to test on Opera Mini.

It was hard to diagnose at the time with the duplicate events, but my theory was that React hydration played a part with react-helmet. The way it behaves is it renders on the server once, then remounts the `<Helmet>` tags again on hydration. Lite skips out hydration, so _in theory_ it shouldn't.

Its just hard to know how Opera Mini servers behave with this though.

Reply:
We could literally do the exact same thing as we've done with Opera Mini and track if its been sent already in the `window` object. I don't _think_ there is a problem with that on Lite, so maybe to be absolutely sure, we just copy the Opera Mini implementation?

Reply:
Yeah that would work. I had thought that we might be able to consolidate both opera mini & lite implementations into a single script, but I think that would make things too complicated (and it is already complicated enough! 😰 )

Reply:
Yea I'd prefer to keep them separate even if they're almost the same. The Lite version may be extended in the future to handle click events too.